### PR TITLE
merge: misha/feat/update-styling-and-ui into master

### PR DESCRIPTION
**Pull request by Splagon**

### CODE REVIEW

File: gfc/financial_companion/templates/pages/individual_group.html [hunk 1]
```diff
@@ -1,80 +1,54 @@
...
+  <li class="list-group-item"><strong>Number of members: </strong> {{count}}</li>
+{% endblock %}
 
-<br>
+{% block table_footer %}
+  If you have any questions regarding the group please contact the owner at: <strong>{{group.owner_email}}</strong>
+{% endblock %}
 
-<div style="text-align:center">
-  <a href="{% url 'individual_group' pk=group.id leaderboard=True %}"><button type="button" class="btn btn-primary px-3 mx-2 py-1 mb-md-4">View Leaderboard</button></a>
```
Comment (charleshss) [inline gfc/financial_companion/templates/pages/individual_group.html:56]: These buttons are used for the users to view their leaderboard. These need to be added to the update view. @Splagon

File: gfc/financial_companion/tests/views/test_all_groups_view.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def test_post_when_partial_group_name_is_applied(self):
         self.assertEqual(len(messages_list), 0)
         self.assertContains(response, "Saving Club")
         self.assertContains(response, "Save with us")
-        self.assertContains(response, "SavingsRUs")
+        self.assertContains(response, "SavingsRUs".title())
```
Comment (charleshss) [inline gfc/financial_companion/tests/views/test_all_groups_view.py:50]: What happens if the user wants to have the name of their group displayed as they wrote it? @Splagon

**Review by charleshss — COMMENTED** [review]
Please review these changes

**Review by MPalmer8 — COMMENTED** [review]
When clicking on the group image in individual_group.html, can you make it so that the image opens in a new tab. Thank you

**Review by MPalmer8 — COMMENTED** [review]
Seed saving accounts. Thank you

File: gfc/financial_companion/templates/pages/individual_transaction.html [hunk 3]
```diff
@@ -1,51 +1,36 @@
...
+  {% endif %}
+  <li class="list-group-item"><strong>Amount:  </strong>{{transaction.currency.upper}} {{transaction.amount}}</li>
+  <li class="list-group-item"><strong>Sender Account:  </strong>{{transaction.sender_account.name.title}}</li>
+  <li class="list-group-item"><strong>Receiver Account:  </strong>{{transaction.receiver_account.name.title}}</li>
+  <li class="list-group-item"><strong>Time of Transaction:  </strong>{{transaction.time_of_transaction}}</li>
 {% endblock %}
+
+{% block additional_items %}{% include "partials/view_models/model_photo.html" with picture=transaction.image name=transaction.title.title%}{% endblock %}
+
+{% block view_model_objects_URL %}{% url 'dashboard' %}{% endblock %}
```
Comment (MPalmer8) [inline gfc/financial_companion/templates/pages/individual_transaction.html:34]: Please make it so that it redirects you to the view transactions page instead of dashboard. Thank you

File: gfc/financial_companion/templates/pages/individual_group.html (hunk 1 above)
Comment (Splagon) [reply gfc/financial_companion/templates/pages/individual_group.html:56]: I see that this error has been made during partialisation and has been swiftly resolved.

File: gfc/financial_companion/tests/views/test_all_groups_view.py (hunk 2 above)
Comment (Splagon) [reply gfc/financial_companion/tests/views/test_all_groups_view.py:50]: I can now foresee the results of my actions and have taken action to remove capitalisation from the individual category/account/transaction/group views

File: gfc/financial_companion/templates/pages/individual_transaction.html (hunk 3 above)
Comment (Splagon) [reply gfc/financial_companion/templates/pages/individual_transaction.html]: I do apologise for this frivolous error and have resolved this issue.

File: gfc/financial_companion/tests/views/test_individual_group_view.py [hunk 4]
```diff
@@ -98,22 +98,6 @@ def test_valid_get_view_individual_group_when_logged_in_as_member(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "pages/individual_group.html")
         group: Group = response.context["group"]
-        self.assertTrue(isinstance(group, UserGroup))
-        self.assertContains(response, self.group.name.capitalize())
```
Comment (charleshss) [inline gfc/financial_companion/tests/views/test_individual_group_view.py:102]: Why was these removed?

**Review by charleshss — APPROVED** [review]
